pynest/nest/lib/hl_api_simulation.py

```diff
@@ -49,6 +49,10 @@
 def Simulate(t):
     """Simulate the network for `t` milliseconds.
 
+    `Simulate` triggers `Prepare`, `Run`, and `Cleanup` following this formula:
+
+    Simulate(t): t' = t/m; Prepare(); for _ in range(m): Run(t'); Cleanup()
```

Comment:
```suggestion
    `Simulate(t)` runs `Prepare()`, `Run(t)`, and `Cleanup()` in this order.
```

Reply:
Also add `Prepare, Run, Cleanup` to the "See also" section below; I cannot add it as a suggestion, unfortunately.